CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
+## (Unreleased)
+* Drop support 1.9.x 2.0.0
```

Comment:
You can say "Drop support of ruby 1.9.x and 2.0.0.". Don't miss period at the end of sentence.

And I also found we mention ruby version in README, so let's change it.